test/arrays/diten/replaceArrayAccess/modifyInAnotherTask.bad

```diff
@@ -0,0 +1,2 @@
+1 2 3 4
```

Comment:
I think you meant to add this to `modifyInAnotherTask2`, looks like you missed the 2 in the file name

Reply:
Yes I definitely meant `modifyInAnotherTask2`, thank you!